app/Services/ContestManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Gladiator\Services;
+
+use Gladiator\Repositories\UserRepositoryContract;
+
+class ContestManager
```

Comment:
@sbsmith86 was chatting with @deadlybutter IRL regarding the name for this service. Would just `Manager` be better and it could handle methods for contests, competitions, waitingrooms? or does `ContestManager` work for all those? What do you think?